scenes/act3.md

```diff
@@ -23,21 +23,21 @@ publish("act3",["roofhunter",3]);
 publish("act3",["roofhong",3]);
 ```
 
-h2: *Ah* that hits the spot.
+h2: *Ah* senti come va giù.
 
 ```
 publish("act3",["roofhunter",2]);
 publish("act3",["roofhong",2]);
 ```
 
-r: Sai, kid...
+r: Sai, tesoro...
```

Comment:
"tesoro",  scelta interessante! :)

Reply:
Sì, in effetti non sono un fan della mia stessa decisione... ma usare "ragazzino" sarebbe stato maschile, quindi "Tesoro" rispetta il genderless dell'originale. Si accettano suggerimenti!